src/Hng.Web/Controllers/UserController.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Hng.Domain.Models;
+using Hng.Infrastructure.Context;
+using Hng.Web.Data;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Hng.Web.Controllers
+{
+    [ApiController]
+    [Route("api/v1/user/")]
+    public class UserController : ControllerBase
+    {
+        private readonly MyDBContext myDBContext;
+        private readonly IMapper mapper;
+
+        public UserController(MyDBContext myDBContext, IMapper mapper)
+        {
+            this.myDBContext = myDBContext;
+            this.mapper = mapper;
+        }
+
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetUserById(int id)
+        {
+            var data = await myDBContext.Users
```

Comment:
@davideshett What is this hack I'm seeing?
Create a generic Repository, using the [Generic Repository pattern](https://learn.microsoft.com/en-us/aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-the-repository-and-unit-of-work-patterns-in-an-asp-net-mvc-application#create-a-generic-repository)